### PR TITLE
Sort asc command

### PR DESCRIPTION
**Pull request by Heixier**
I added an unnecessary ft_split feature into it. This adds the sort_asc command which takes a given string of whitespace-separated integers and sorts them in ascending order. Rayden and I have tested it and it passed our tests.

### CODE REVIEW

**Comment by Heixier** [comment]
Both Baixier and Heixier are me, I'm logged into two different accounts on my computer and it's a bit messy

**Comment by tanamaroby** [comment]
> Both Baixier and Heixier are me, I'm logged into two different accounts on my computer and it's a bit messy

-42 for cheating and having outside git commit

File: src/commands/ping.ts [hunk 1]
```diff
@@ -23,7 +27,6 @@ export const data = new SlashCommandBuilder()
  * Executes the "ping" command.
  * @param {CommandInteraction} interaction - The interaction event triggered by the command.
  */
-export const execute = async (interaction: CommandInteraction) => {
-  // Reply to the interaction with "PONG!"
-  interaction.reply("PONG!");
+export const execute = async (interaction: ChatInputCommandInteraction) => {
+  const message = interaction.options.getString("message", true);
```
[the remark this answers is not in the document]
Comment (Heixier) [reply src/commands/ping.ts:31]: lmao so that's where it was

**Comment by tanamaroby** [comment]
Don't forget to increase the version in package.json!

**Review by tanamaroby — APPROVED** [review]
The LGest of the TM!

**Pull request merged**